packages/mui-base/src/Field/Control/useFieldControlValidation.ts

```diff
@@ -4,9 +4,17 @@ import { useEventCallback } from '../../utils/useEventCallback';
 import { useFieldRootContext } from '../Root/FieldRootContext';
 import { mergeReactProps } from '../../utils/mergeReactProps';
 import { DEFAULT_VALIDITY_STATE } from '../utils/constants';
+import { useFormRootContext } from '../../Form/Root/FormRootContext';
+import { getCombinedFieldValidityData } from '../utils/getCombinedFieldValidityData';
 
 const validityKeys = Object.keys(DEFAULT_VALIDITY_STATE) as Array<keyof ValidityState>;
 
+/**
+ *
+ * API:
+ *
+ * - [useFieldControlValidation API](https://mui.com/base-ui/api/use-field-control-validation/)
+ */
```

Comment:
Hooks are no longer documented, so this URL doesn't exist. The current `docs:api` script doesn't annotate hooks this way anymore, but if you copied files from an older version, you'll have to manually remove these comments from hooks.